br/tests/config/tikv.toml

```diff
@@ -37,5 +37,5 @@ path = "/tmp/backup_restore_test/master-key-file"
 [log-backup]
 max-flush-interval = "50s"
 [gc]
-ratio-threshold = 1.1
+ratio-threshold = -1.0
```

Comment:
it is necessary?

Reply:
every time I run the test locally it will create this item, need to dig into it, somehow didn't get reverted in the code